client/src/components/annotators/useMediaController.ts

```diff
@@ -71,7 +71,12 @@ export default function useMediaController({ emit }: {
       right: params.map.maxBounds.right,
       bottom: params.map.maxBounds.bottom,
     });
-    geoViewerRef.value.zoomRange(params.map);
+    geoViewerRef.value.zoomRange({
+      // do not set a "zoom out" limit so that 1x(map size) is always min.
+      min: -Infinity,
```

Comment:
This seems to have the desired effect: always make min zoom the size of the map.

I can't figure out _why_ it does this, but I figured @BryonLewis will know.